src/controllers/certificateSigned.controller.ts

```diff
@@ -0,0 +1,16 @@
+
+import { controller, httpGet } from 'inversify-express-utils'
+
+import BaseController from 'app/controllers/base.controller'
+import { CERTIFICATE_SIGNED_URI } from 'app/paths'
+import TYPES from 'app/types'
+
+@controller(CERTIFICATE_SIGNED_URI, TYPES.SessionMiddleware, TYPES.AuthMiddleware, TYPES.CompanyAuthMiddleware)
+export class CertificateSignedController extends BaseController {
+
+  @httpGet('')
+  public async get(): Promise<string> {
+
```

Comment:
I've noticed this in a few places - why the new line at the start of the method?